Tests/PINAnimatedImageTests.swift

```diff
@@ -132,4 +132,27 @@ class PINAnimatedImageTests: XCTestCase, PINRemoteImageManagerAlternateRepresent
         }
         self.waitForExpectations(timeout: self.timeoutInterval(), handler: nil)
     }
+    
+    func testGIFBytes() {
+        let animatedExpectation =  self.expectation(description: "Animated image should be downloaded")
+        let imageManager = PINRemoteImageManager.init(sessionConfiguration: nil, alternativeRepresentationProvider: self)
+        imageManager.downloadImage(with: self.slowAnimatedGIFURL()!) { (result : PINRemoteImageManagerResult) in
+            XCTAssert(result.image == nil)
+            guard let animatedData = result.alternativeRepresentation as? NSData else {
+                XCTAssert(false, "alternativeRepresentation should be able to be coerced into data")
+                return
+            }
+            
+            XCTAssert(animatedData.pin_isGIF() && animatedData.pin_isAnimatedGIF())
+            
+            let pinCachedAnimatedImage = PINGIFAnimatedImage(animatedImageData: animatedData as Data)!
+            let bytesSize = UInt32((pinCachedAnimatedImage.image(at: 0, cacheProvider: nil)!.takeUnretainedValue() as CGImage).bytesPerRow) * pinCachedAnimatedImage.height
+
+            XCTAssert(pinCachedAnimatedImage.bytesPerFrame <= bytesSize)
```

Comment:
Should this be == ? I think this test succeeds with the old bytesPerFrame

Reply:
Thanks for adding a test! As @garrettmoon mentions, please ensure the test fails without the code change.

Reply:
In theory, `pinCachedAnimatedImage.bytesPerFrame` should always `<=` `bytesSize`, because `CGBitmapContextCreate` may add padding and alignment when allocated memory, but in this sample url, seems we can use `==`. Let's try it. :)